docs/list.md

```diff
@@ -150,27 +160,6 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
     - After they have merged your pull request(s), you can claim your swag.
 - **Notes**: More details for the Hackathon prizes and Hacktoberfest details [are here](https://medusajs.com/blog/medusa-hackathon){:target="\_blank"}.
 
-#### **Meilisearch**
-
-- **Swag**: Reusable Bottles, Stickers (possibly? Awaiting updated swag for 2022.)
-- **Requirements**:
-    - Submit 1 or more pull requests to qualify as a contributor. Check out the [Contributing Guidelines](https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md) for more details.
-- **How to sign up**: Join the [Slack community](https://www.notion.so/6cd1509802984122ab74745f637b694f) and the [#hacktoberfest channel](https://meilicommunity.slack.com/archives/C02G1H43J2X) to connect with Meilisearch employees and for any queries.
-- **Issues**: See the list of [hacktoberfest-labeled issues](https://github.com/search?q=org%3Ameilisearch+label%3Ahacktoberfest&state=open&type=Issues).
-- **Notes**: Check out the [blog post](https://blog.meilisearch.com/contribute-hacktoberfest-2022/) for more info.
```

Comment:
Hi @souravjain540! 👋 Thank you for letting me know about Amplication! I'd love to get you all on the Swag List ASAP! Can you please rebase or re-add Meilisearch here? It seems like it's being accidentally deleted.

Thanks! 👍